Resources/public/js/views/services/ez-trashviewservice.js

```diff
@@ -223,6 +223,54 @@ YUI.add('ez-trashviewservice', function (Y) {
         },
 
         /**
+         * Restores trash items
+         *
+         * @method _restoreItems
+         * @protected
+         * @param {Object} e restoreItems event facade
+         * @param {Array} e.trashItems  List of trashItems to be restored
+         */
+        _restoreItems: function (e) {
+            var loadOptions = {api: this.get('capi')},
+                tasks = new Y.Parallel(),
+                service = this,
+                notificationNamePrefix = "restoreTrashItem-";
+
```

Comment:
that's maybe a bit of niptick but i think you could have even fired a started notification here in addition of the ones that are in your restore() callback